examples/meta/src/meta_api/enums.sg

```diff
@@ -0,0 +1 @@
+enum LIBLINEAR_SOLVER_TYPE.L2R_L2LOSS_SVC
```

Comment:
Hey, I just had another look at this. I'm pretty sure the statement `LIBLINEAR_SOLVER_TYPE myVar = enum LIBLINEAR_SOLVER_TYPE.L2R_L2LOSS_SVC` is going to work in all languages without any modifications to the translation rules.